examples/trusted_information_retrieval/backend/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "trusted_information_retrieval_backend"
+version = "0.1.0"
+authors = ["Ivan Petrov <ivanpetrov@google.com>"]
+edition = "2018"
+license = "Apache-2.0"
+
+[dependencies]
+anyhow = "*"
+env_logger = "*"
+futures = "*"
+futures-core = "*"
+futures-util = "*"
+log = "*"
+prost = "*"
+reqwest = "*"
+serde = { version = "*", features = ["derive"] }
+structopt = "*"
+tokio = { version = "*", features = [
+  "fs",
+  "macros",
+  "rt-threaded",
+  "sync",
+  "stream"
+] }
+tonic = { version = "*", features = ["tls"] }
+quick-xml = { version = "*", features = ["serialize"] }
```

Comment:
Why do we need the serialize feature? It seems we only need to deserialize.

Reply:
It's for both serializing and deserializing:
https://docs.rs/quick-xml/0.18.1/quick_xml/#features